app/resources/v1/ont/library_resource.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
```

Comment:
Again wondering whether there is an opportunity to share any behaviour? I am not sure especially as it is volatile.

Reply:
There probably is an opportunity to abstract the shared behaviour but as you said the models may change in the coming weeks. Can have a look back and refactor after the work is solid.

Reply:
Agreed.